### PR TITLE
Write a the spanish translation of docs\community\starters.md

### PR DESCRIPTION
**Pull request by AlexScigalszky**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe: Write a the spanish translation of docs\community\starters.md

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by SanderElias — DISMISSED** [review]
LGTM, but as I don't understand spanish, I'm going to ask @aaronfrost to check the content

**Comment by Villanuevand** [comment]
@SanderElias  I'll take a look...

File: docs/community/starters_es.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+En esta página tenemos la lista de proyectos que fueron creados por nuestra comunidad.
+
+<details>
+<summary>Scully, Flotiq y Snipcart boilerplate</summary>
+
+## Scully, Flotiq y Snipcart boilerplate
+
+Crea un sitio de venta online con Scully, Snipcart y Flotiq.
+
+Obtendra un sitio para ventas online simple y funcional con todos tus productos, ejecutandose en Internet.
```
Comment (Villanuevand) [inline docs/community/starters_es.md:19]:
```suggestion
Obtén un sitio para ventas online simple y funcional con todos tus productos, ejecutandose en Internet.
```

File: docs/community/starters_es.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Utiliza Flotiq Angular SDK, que simplifica el desarrollo!
+
+Con unos pocos clics puedes deployar tu proyecto en Netlify!
```
Comment (Villanuevand) [inline docs/community/starters_es.md:23]:
```suggestion
Con unos pocos clics puedes desplegar tu proyecto en Netlify!
```

File: docs/community/starters_es.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+<details>
+<summary>Event Calendar con Scully y Flotiq</summary>
+
+## Event Calendar con Scully y Flotiq
+
+Construye tu propio calendario y completalo con eventos que puedes compartir con otros!
+
+Utiliza Flotiq Angular SDK, que simplifica el desarrollo!
+
+Con unos pocos clics puedes deployar tu proyecto en Netlify!
```
Comment (Villanuevand) [inline docs/community/starters_es.md:38]:
```suggestion
Con unos pocos clics puedes desplegar tu proyecto en Netlify!
```

File: docs/community/starters_es.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+[Event Calendar con Scully y Flotiq](https://github.com/flotiq/scully-event-calendar-starter)
+
+</details>
+
+<details>
+<summary>Scully y Flotiq para blog de cocina</summary>
+
+## Scully y Flotiq para blog de cocina
+
+Confligura tu blog de cocina en unos simples pasos y comparte esas maravillosas recetas con el mundo!
```
Comment (Villanuevand) [inline docs/community/starters_es.md:49]:
```suggestion
Configura tu blog de cocina en algunos simples pasos y comparte tus maravillosas recetas con el mundo!
```

File: docs/community/starters_es.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+Utiliza Flotiq Angular SDK, que simplifica el desarrollo!
+
+Con unos pocos clics puedes deployar tu proyecto en Netlify!
```
Comment (Villanuevand) [inline docs/community/starters_es.md:53]:
```suggestion
Con unos pocos clics puedes desplegar tu proyecto en Netlify!
```

File: docs/community/starters_es.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+[Scully y Flotiq para blog de cocina](https://github.com/flotiq/scully-recipes-starter)
+
+</details>
+
+<details>
+<summary>Scully y Flotiq - portfolio</summary>
+
+## Scully y Flotiq - portfolio
+
+Crea tus proyectos con Sculy y Flotiq y muestralos con todas las personas del alrededor del mundo!
```
Comment (Villanuevand) [inline docs/community/starters_es.md:64]:
```suggestion
Crea tus proyectos con Scully y Flotiq y muéstralos
 a otros alrededor del mundo!
```

File: docs/community/starters_es.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+Utiliza Flotiq Angular SDK, que simplifica el desarrollo!
+
+Con unos pocos clics puedes deployar tu proyecto en Netlify!
```
Comment (Villanuevand) [inline docs/community/starters_es.md:68]:
```suggestion
Con unos pocos clics puedes desplegar tu proyecto en Netlify!
```

File: docs/community/starters_es.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+<details>
+<summary>Scully y Flotiq - blog</summary>
+
+## Scully y Flotiq - blog
+
+Construye tu blog personal en unos pocos pasos usando Scully y Flotiq!
+
+Utiliza Flotiq Angular SDK, que simplifica el desarrollo!
+
+Con unos pocos clics puedes deployar tu proyecto en Netlify!
```
Comment (Villanuevand) [inline docs/community/starters_es.md:83]:
```suggestion
Con unos pocos clics puedes desplegar tu proyecto en Netlify!
```

**Review by Villanuevand — CHANGES_REQUESTED** [review]
@AlexScigalszky
I just made some comments about words to change...

Let me know anything else ... 
Great work here, thanks for the support.

**Comment by AlexScigalszky** [comment]
@Villanuevand  thanks it looks better!

**Review by Villanuevand — APPROVED** [review]
LGTM! 

Thanks @AlexScigalszky  for your support. 

To get this PR merged you need to update this branch.

**Pull request merged**